### ISSUE TITLE
Error in `bento init` following "Python package flake8-json will be installed"

### ISSUE THREAD

**Issue by dmontagu** [returntocorp/bento#71]
**Describe the bug**
MacOS Mojave 10.14.6
Python 3.7.3
Poetry-generated virtual environment

```
> bento init

...

Detected project with Python

✔ Creating virtual environment for r2c.bandit
✔ Python package bandit will be installed
pip3 install -✔ Creating virtual environment for r2c.flake8
-upgrade bento-cli✔ Python package flake8 will be installed
✔ Python package flake8-json will be installed
Traceback (most recent call last):
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/__main__.py", line 14, in main
    cli()
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/cli.py", line 526, in init
    network.post_metrics(bento.metrics.command_metric("setup"))
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/metrics.py", line 111, in command_metric
    "repository": __get_git_url(),
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/metrics.py", line 36, in __get_git_url
    if r and r.remotes and r.remotes.origin:
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/git/util.py", line 889, in __getattr__
    return list.__getattribute__(self, attr)
AttributeError: 'IterableList' object has no attribute 'origin'
✘ There was an exception 'IterableList' object has no attribute 'origin'

>
```

Not sure what, if any, additional output I should be expecting if that error hadn't occurred.

**Comment by dmontagu** [comment]
I tried running `bento check` after installing just for fun and got this issue:

```
bento check                     

╭─────────────────────────────────────────────╮
│  🎉 A new version of Bento is available 🎉  │
│  Try it out by running:                     │
│                                             │
│       pip3 install --upgrade bento-cli      │
│                                             │
╰─────────────────────────────────────────────╯

Running Bento checks...
  r2c.bandit: |███████████████| 00:06, 🍱
  r2c.flake8: |███████████████| 00:05, 🍱
                                                 
Traceback (most recent call last):               
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/__main__.py", line 14, in main
    cli()
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/cli.py", line 630, in check
    tool_id, findings, get_ignores_for_tool(tool_id, config)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/metrics.py", line 103, in violations_to_metrics
    for aggregates in __get_aggregate_violations(violations)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/metrics.py", line 103, in <listcomp>
    for aggregates in __get_aggregate_violations(violations)
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/bento/metrics.py", line 36, in __get_git_url
    if r and r.remotes and r.remotes.origin:
  File "/Users/dmontague/Library/Caches/pypoetry/virtualenvs/app--3J4JXvL-py3.7/lib/python3.7/site-packages/git/util.py", line 889, in __getattr__
    return list.__getattribute__(self, attr)
AttributeError: 'IterableList' object has no attribute 'origin'
✘ There was an exception 'IterableList' object has no attribute 'origin'

**Comment by dlukeomalley** [comment]
I've marked this issue as `priority:high` because it's workflow blocking - it's not possible to use Bento with this bug in projects that don't have an origin remote.

**Comment by dmontagu** [comment]
Adding a remote called "origin" fixed the problem. Thanks!

**Comment by dlukeomalley** [comment]
This fix has made it into `bento/0.1.0.post1`. @dmontagu, let us know if the issue persists for you. Thank you for the issue 🙏
